code/game/sound.dm

```diff
@@ -55,11 +55,8 @@ var/global/const/FALLOFF_SOUNDS = 0.5
 
 	volume *= pressure_factor
 
-	if(istype(T,/turf/simulated) && istype(turf_source,/turf/simulated))
-		var/turf/simulated/sim_source = turf_source
-		var/turf/simulated/sim_destination = T
-		if(sim_destination.zone != sim_source.zone)
-			volume -= 30
+	if((T.zone || turf_source.zone) && T.zone != turf_source.zone)
+		volume -= 30
```

Comment:
why does this exist at all? isnt this what pressure factor is for

Reply:
No that's for things being separated by airtight stuff